README.md

```diff
@@ -16,29 +16,43 @@ Photos to use for testing:
 CONOPS/Rules:
 * [2015 CONOPS/Rules](https://drive.google.com/open?id=0BySpWXvmBM4JajdsemZFb0o2Ukk&authuser=0)
 * [2016 CONOPS/Rules](https://drive.google.com/open?id=0BySpWXvmBM4JUnhyaGN0NW5OR1k)
+* [2017 CONOPS/Rules](https://drive.google.com/file/d/0BySpWXvmBM4JNlFyX18xTUpPOG8/view?usp=sharing)
 
 ### Building and Installing
-Well... seeing as there isn't actually any source in the repository yet, this isn't actually possible at the moment
-
-When we have actually implemented stuff the following will probably apply
 
 **Build system**: CMake to set up build environment then whatever CMake gives you (depends on platform, make for Linux, etc.)
 
 **Dependencies**: 
-    [OpenCV 3.0](http://opencv.org/downloads.html)
-    [Boost 1.58](http://www.boost.org/users/download/)
-    [DeckLink](http://www.blackmagicdesign.com/support/sdks)
+    [OpenCV 3.0](http://opencv.org/downloads.html),
+    [Boost 1.58](http://www.boost.org/users/download/),
+    [DeckLink](http://www.blackmagicdesign.com/support/sdks) _(optional)_,
+    [ZBar] (http://zbar.sourceforge.net/download.html) 
 
 ####Installation Instructions:
-1. Clone the Project
-2. Install dependencies
+1. Install the dependencies listed above. This will vary based on your system. 
+2. Clone the Project
 3. Use CMake to setup the project from the CMake list
 4. Compile
 
+For Linux, refer to the detailed instructions here: [Linux Installation](http://docs.uwarg.com/computer-vision/Building-the-project-[Linux]/)
```

Comment:
Ok. I cleaned it up a bit more.
I combined 3.+4., but honestly the list is so short, I don't know if I should just remove it.